### ISSUE TITLE
`install_components` must be a comma-separated string, not a list

### ISSUE THREAD

**Issue by kkroening** [google-github-actions/setup-gcloud#471]
### TL;DR

[The readme states](https://github.com/google-github-actions/setup-gcloud) that `install_components` should be a "List of Cloud SDK components to install" but it appears that a comma-separated string is expected rather than a list.

### Expected behavior

I would expect to be able to use a YAML list to specify the `install_components` input.

### Observed behavior

If a YAML list is used to specify the `install_components` input such as `install_components: [beta, cloud-datastore-emulator]`, the following error is produced:
```
The workflow is not valid. .github/workflows/test.yaml (Line: 17, Col: 9): A sequence was not expected
```
But using a comma-separated string such as `install_components: beta, cloud-datastore-emulator` works.

### Action YAML

```yaml
name: Test
on: [push]
jobs:
  test:
    runs-on: ubuntu-20.04
    steps:
    - name: Set up Cloud SDK
      uses: google-github-actions/setup-gcloud@v0
      with:
        install_components:
        - beta
        - cloud-datastore-emulator
```

### Additional information

Perhaps this is expected behavior, but if so it seems to contradict what's written in the readme.

**Comment by sethvargo** [comment]
I don't think GitHub Actions allows for anything other than a string unfortunately. We need to update the docs on this.

**Comment by Ido-Ozeri** [comment]
I was also baffled with the input convention for `install_components` and the documentation is very obscure about it. An example of more than 1 component would be highly appreciated in the docs and could prevent future confusion.

**Comment by mattwelke** [comment]
Example in the docs is good. Started using this action today. How to install components was clear to me. I'm having an issue using my installed component (pubsub emulator) but that's another issue.
